.github/workflows/manual_release.yml

```diff
@@ -0,0 +1,113 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual Maven Release
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      comment:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Comment for release'
+        # Default value if no value is explicitly provided
+        default: 'Start release from WEB UI'
+        # Input has to be provided for the workflow to run
+        required: true
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  print_params:
+    name: Print parameters
+
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+
+    # Runs a single command using the runners shell
+    - name: Print
+      run: |
+        echo "github.event.inputs.comment ${{ github.event.inputs.comment }}"
+        echo "github.ref: ${{ github.ref }}"
+        echo "github.event.ref: ${{ github.event.ref }}"
+        echo "github.repository: ${{ github.repository }}"
+        echo "github.workspace: ${{ github.workspace }}"
+  release_job:
+    name: Maven release
+
+    # Permission ellenorzes https://github.community/t/who-has-permission-to-workflow-dispatch/133981
+#    if: contains('["rombow","czenczl"]', github.actor)
```

Comment:
Ez nem kell bele?

Reply:
a modul csapat fogja releaselni, de elosszor azt is be kell allitani